shared_code/utils.py

```diff
@@ -10,7 +10,8 @@
 
 client = MongoClient(
     "mongodb+srv://" + os.environ['DBUSERNAME'] + ":" + os.environ['PW']
-    + "@shelter-rm3lc.azure.mongodb.net/shelter?retryWrites=true&w=majority"
+#    + "@shelter-rm3lc.azure.mongodb.net/shelter?retryWrites=true&w=majority"
```

Comment:
You need to uncomment this and remove yours.

Reply:
I set the database to that of ShelterApp.

Reply:
Well it's giving me Authentication Failed exception with username and password which automated scrapers are designed to use.  Why is the change in connection string needed btw?